source/manual/rename-a-country.html.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Rename a country
+parent: "/manual.html"
+layout: manual_layout
+section: Publishing
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2018-06-19
+review_in: 6 months
+related_applications: [travel-advice-publisher, content-tagger, whitehall]
+---
+
+We recommend testing out these instructions in integration and then staging before using them in production.
+
+### 1. Update Foreign Travel Advice publisher
+
+This will update www.gov.uk/foreign-travel-advice/countryname. In [Travel Advice Publisher](https://github.com/alphagov/travel-advice-publisher):
+
+1. Create a pull request with:
+
+* A migration to update the relevant `country_slug` of TravelAdviceEdition. [Example] (https://github.com/alphagov/travel-advice-publisher/pull/346/commits/b28ff7b4eae96543324f61be700dca32f1ffdba5)
```

Comment:
The `Example` link doesn't render as expected, is there whitespace between `[Example]` and `(https://github.com/alphagov/travel-advice-publisher/pull/346/commits/b28ff7b4eae96543324f61be700dca32f1ffdba5)` ?